public/stylesheets/landing.css

```diff
@@ -2,13 +2,21 @@ body {
   background-color: #000;
 }
 
+h1 {
+  padding-bottom: 15px;
+}
+
 #landing-header {
   z-index: 1;
   position: relative;
   text-align: center;
   padding-top: 40vh;
 }
 
+#landing-form {
```

Comment:
It's more preferred to style elements using classes, not IDs.
https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4